.gitignore

```diff
@@ -41,3 +41,6 @@ htmlcov/
 *.tfvars
 *.tfstate.backup
 *.tfstate.*.backup
+.vscode
+.env
+reports/
```

Comment:
Also exclude virtualenv folders like `venv/` and `.venv/`?

Reply:
Thanks, sgtm !